spec/bump_spec.rb

```diff
@@ -403,7 +403,7 @@ module Foo
         gemspec
       RUBY
       `git add Gemfile #{gemspec}`
-      Bundler.with_clean_env { run("bundle") }
+      Bundler.with_unbundled_env { run("bundle") }
```

Comment:
use the same logic here ?
... otherwise do a `gem install bundler`
... can drop ruby 2.3 / 2.4 to avoid a few headaches

Reply:
Pushed new commit with same logic for specs.

Not my call about dropping older version of ruby :)